src/app/server-cluster/DefaultServerCluster.h

```diff
@@ -64,6 +67,13 @@ class DefaultServerCluster : public ServerClusterInterface
     /// Default implementation just returns the global attributes required by the API contract.
     CHIP_ERROR Attributes(const ConcreteClusterPath & path, ReadOnlyBufferBuilder<DataModel::AttributeEntry> & builder) override;
 
+    /// Must only be implemented if event readability is relevant
+    CHIP_ERROR EventInfo(const ConcreteEventPath & path, DataModel::EventEntry & eventInfo) override
+    {
+        eventInfo.readPrivilege = Access::Privilege::kView;
```

Comment:
As a followup, I wonder whether we could use generated metadata here... I worry about people forgetting to implement this thing for events that do not in fact have View privilege in the spec; nothing catches that if they do, right?

Reply:
created #39390